dist/index.html

```diff
@@ -32,8 +32,6 @@ <h1>Overlook</h1>
           <input class="date-input" type="date" name="Input date" />
           <label for="room-type">Room Type
             <select class="type-select" name="room-type" id="room-type">
-              <option value="none">None</option>
-              <option value="one-bed">One Bed</option>
             </select>
```

Comment:
I'm curious why you removed the options for this select? Did you mean to replace them?

Reply:
Ahhh that makes sense! Good work!